app/controllers/adminpanel_controller.rb

```diff
@@ -0,0 +1,47 @@
+# Administration Panel
+# frozen_string_literal: false
+
+# Administration user and administration panel
+class AdminpanelController < ApplicationController
```

Comment:
Not sure is the best have a same controller for users and for admin panel. I think will be better have separate controllers for each one

Reply:
I tried to add a controller called User but when I wanted to enter the login path it showed me all blank for the user registration as well.